app/controllers/mod_warning_controller.rb

```diff
@@ -44,8 +44,8 @@ def create
                               body: params[:mod_warning][:body], is_suspension: params[:mod_warning][:is_suspension],
                               suspension_end: suspension_end, active: true)
     if @warning.save
-      if params[:mod_warning][:is_suspension]
-        @user.community_user.update(is_suspended: true, suspension_end: suspension_end,
+      if !!params[:mod_warning][:is_suspension]
+        @user.community_user.update(is_suspended: params[:mod_warning][:is_suspension], suspension_end: suspension_end,
```

Comment:
Should have an `!!` prepended to it too; if that's the condition, it should be the value too, so that we know it's the right type.